editor/src/input/mouse.rs

```diff
@@ -22,6 +22,10 @@ impl ViewportBounds {
 	pub fn size(&self) -> DVec2 {
 		self.bottom_right - self.top_left
 	}
+
+	pub fn mid(&self) -> DVec2 {
```

Comment:
to stay somewhat consistent with the existing code I would prefer center

Reply:
Done